etc/opensc.conf.in

```diff
@@ -373,11 +374,15 @@ app default {
 		# notify_pin_bad_text = "bad text";
 	# }
 
-	# PIV cards need an entry similar to this one:
-	# card_atr 3B:7D:96:00:00:80:31:80:65:B0:83:11:00:AC:83:00:90:00 {
-		# name = "PIV-II";
-		# driver = "piv";
-	# }
+	# Yubikey NEO is known to have the PIV applet and the OpenPGP applet
```

Comment:
It's not only Yubikey Neo anymore. This functionality is present in the recent Yubikey 4. Mine is `Yubico Yubikey 4 OTP+U2F+CCID` with ATR `3B F8 13 00 00 81 31 FE 15 59 75 62 69 6B 65 79 34 D4`.
Not sure if it matters to have this ATR in default configuration, or there should be both/all we will be able to gather.

Reply:
actually I've extracted this information from the OpenPGP driver. There are ATRs here https://github.com/Yubico/yubico-piv-tool/blob/3aaa525efc678f9bca1fb08716e282eb00536356/lib/ykpiv.h#L672-L673 and here http://ludovic.rousseau.free.fr/softwares/pcsc-tools/smartcard_list.txt. I'll integrate them later, thanks for the pointer.